types/v56/openapi.go

```diff
@@ -50,3 +50,44 @@ type Role struct {
 	BundleKey   string `json:"bundleKey"`
 	ReadOnly    bool   `json:"readOnly"`
 }
+
+type VdcComputePolicy struct {
```

Comment:
Maybe here a comment that this is sizing policy in UI?

Reply:
added